src/include/daos_srv/pool.h

```diff
@@ -56,6 +56,8 @@ struct ds_pool {
 	ABT_mutex		sp_iv_refresh_lock;
 	struct ds_iv_ns	       *sp_iv_ns;
 	uint32_t		sp_dtx_resync_version;
+	uuid_t			sp_srv_cont_hdl;
```

Comment:
is it the server container handle for rebuild and EC aggregate, so there is only one container in process at the same time?
some comments might be helpful, thx

Reply:
Agreed.

Reply:
ok